apps/comps/components/agent-spotlight-section.tsx

```diff
@@ -16,6 +17,7 @@ export const AgentSpotlightSection: React.FC<AgentSpotlightSectionProps> = ({
 }) => {
   return (
     <section className="my-12">
+      <TermsModal />
```

Comment:
not that it really matters at this point, but is there a reason to place this here vs importing it directly in `apps/comps/app/competitions/page.tsx`?